src/v2/Apps/Conversation/Components/Message.tsx

```diff
@@ -39,6 +40,15 @@ const AttachmentContainer = styled(Flex)<
   justify-content: space-between;
 `
 
+const MessageText = styled(Sans)`
+  white-space: pre-line;
+  && {
+    a:hover {
+      color: currentcolor;
```

Comment:
TIL I can simply use `currentcolor` to avoid needing to conditionally define desired color again.